platform/generic/thead/thead-generic.c

```diff
@@ -14,6 +14,7 @@
 #include <sbi/sbi_string.h>
 #include <sbi_utils/fdt/fdt_helper.h>
 
+#define CSR_MCOUNTERWEN  0x7c9
```

Comment:
Just use definition in thead_c9xx.h, we do not need to define this csr by ourselves.